src/Components/ContactList/ContactsList.js

```diff
@@ -1,25 +1,42 @@
-import React from 'react';
-import { FlatList } from 'react-native';
+import React, { useMemo } from 'react';
+import { SectionList, Text, View, StyleSheet } from 'react-native';
 
-import MOCK_DATA from '../../../data/MOCK_DATA.json';
+import contactListInMap from '../../Utils/contactListWithTitle';
+import colors from '../../Constants/colors';
 
 import Contact from './Contact';
 
-const ContactsList = () => (
-  <FlatList
-    data={MOCK_DATA}
-    keyExtractor={item => item.id}
-    renderItem={({ item }) => {
-      return (
-        <Contact
-          name={item?.name}
-          surname={item?.surname}
-          image={item?.avatar_image}
-          phoneNumber={item?.phone_number}
-        />
-      );
-    }}
-  />
-);
+const ContactsList = () => {
+  const data = useMemo(() => contactListInMap(), []);
+  return (
+    <SectionList
+      sections={data}
+      keyExtractor={item => item.id}
+      renderItem={({ item }) => {
+        return <Contact item={item} />;
```

Comment:
Please rename "item" props to contact.
Contact component waits for "contact". not abstract "item"